client/src/Movies/Movie.js

```diff
@@ -1,54 +1,45 @@
 import React, { useState, useEffect } from 'react';
 import axios from 'axios';
+import { useParams } from 'react-router-dom'
+
+import MovieCard from './MovieCard';
+import MovieList from './MovieList';
 
 export default function Movie(props) {
+  const { addToSaveList } = props
```

Comment:
Double check the naming of the function you're bringing in here. Is this the _exact_ same name you passed in App.js? (Line 48)